app/src/main/java/org/literacyapp/authentication/MergeThread.java

```diff
@@ -123,32 +123,35 @@ private synchronized void findSimilarStudentsUsingMeanFeatureVector(PreProcessor
         // Iterate through all StudentImageCollectionEvents
         List<StudentImageCollectionEvent> studentImageCollectionEvents = studentImageCollectionEventDao.loadAll();
```

Comment:
For debugging, add the size of the fetched list to the log output: 
`Log.i(getClass().getName(), "studentImageCollectionEvents.size(): " + studentImageCollectionEvents.size());`